Traceback (most recent call last):
  File "/home/jebarrios/anaconda3/envs/montse-tBLG/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 478, in _poll_for_reply
    msg = self.kc.shell_channel.get_msg(timeout=timeout)
  File "/home/jebarrios/anaconda3/envs/montse-tBLG/lib/python3.7/site-packages/jupyter_client/blocking/channels.py", line 57, in get_msg
    raise Empty
_queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/jebarrios/anaconda3/envs/montse-tBLG/lib/python3.7/site-packages/jupyter_cache/executors/basic.py", line 163, in execute
    executenb(nb_bundle.nb, cwd=tmpdirname, timeout=timeout)
  File "/home/jebarrios/anaconda3/envs/montse-tBLG/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 737, in executenb
    return ep.preprocess(nb, resources, km=km)[0]
  File "/home/jebarrios/anaconda3/envs/montse-tBLG/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 405, in preprocess
    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)
  File "/home/jebarrios/anaconda3/envs/montse-tBLG/lib/python3.7/site-packages/nbconvert/preprocessors/base.py", line 69, in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
  File "/home/jebarrios/anaconda3/envs/montse-tBLG/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 438, in preprocess_cell
    reply, outputs = self.run_cell(cell, cell_index, store_history)
  File "/home/jebarrios/anaconda3/envs/montse-tBLG/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 578, in run_cell
    exec_reply = self._poll_for_reply(parent_msg_id, cell, timeout)
  File "/home/jebarrios/anaconda3/envs/montse-tBLG/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 483, in _poll_for_reply
    self._check_alive()
  File "/home/jebarrios/anaconda3/envs/montse-tBLG/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 510, in _check_alive
    raise DeadKernelError("Kernel died")
nbconvert.preprocessors.execute.DeadKernelError: Kernel died
